src/store/transactions/transfer/transfer.reducer.ts

```diff
@@ -194,6 +196,12 @@ function transferReducer(
           error: action.error,
         },
       };
+    case TransferActionTypes.SET_RECEIVER_ACCOUNTS_CREATION_AUTHORIZATION_APPROVAL: {
```

Comment:
I think approval is redundant with authorization

Reply:
right, fixed!